less/styles.less

```diff
@@ -12,8 +12,13 @@
 @redSidebarColor: #BB3C33;
 @footerBackgroundColor: #f1f1f1;
 
+html, body {
+  font-size: 62.5%;
```

Comment:
the whole point of using 62.5% is because we, as developers, have no control on what a user has set their browser default pixel size to—and we respect that. generally, browser default font size is 16px, so that's what we base our calculations off of. if a user has changed their default font size, then the whole page will respond relative to that user's settings.